packages/surveys/src/utils/status-utils.ts

```diff
@@ -0,0 +1,62 @@
+import { IGetSurveyFormJson, IGetSurveySchedule } from "@esri/hub-common";
+
+/**
+ * Parses survey form item and form.json to determine if survey is draft, scheduled, open, or closed
+ * @param {IGetSurveyFormJson} form Form.json item resource associated with survey item
+ * @param {Date} now Date object
+ * @returns {string}
+ */
+export function getSurveyStatus (
+  { settings: { openStatusInfo: { status: _status, schedule } } }: IGetSurveyFormJson,
+  now?: Date
+): string {
+  let status = _status;
+  if (status === 'scheduled') {
```

Comment:
per above suggest of a `SurveyStatus` enum this could then be `if( status === SurveyStatus.scheduled)` which is more typed and may help prevent string bugs.